Polygon_mesh_processing/include/CGAL/Polyhedral_envelope.h

```diff
@@ -276,6 +273,37 @@ struct Polyhedral_envelope {
   Polyhedral_envelope()
   {}
 
+  // Disabled copy constructor & assignment operator
+  Polyhedral_envelope(const Polyhedral_envelope<GeomTraits>&) = delete;
+  Polyhedral_envelope<GeomTraits>& operator=(const Polyhedral_envelope<GeomTraits>&) = delete;
+
+  Polyhedral_envelope<GeomTraits>& operator=(Polyhedral_envelope<GeomTraits>&& other) noexcept
+  {
+    halfspace = std::move(other.halfspace);
+    bounding_boxes = std::move(other.bounding_boxes);
+    env_vertices = std::move(other.env_vertices);
+    env_faces = std::move(other.env_faces);
+    tree = std::move(other.tree);
+    oriented_side = std::move(other.oriented_side);
+
+    const_cast<Tree_traits&>(tree.traits())
+      .set_shared_data(Datum_map<GeomTraits>(bounding_boxes),
+                       Point_map<GeomTraits>(bounding_boxes));
+
+    return *this;
+  }
+
+  Polyhedral_envelope(Polyhedral_envelope<GeomTraits>&& other)
+    : halfspace(other.halfspace)
+    , bounding_boxes(other.bounding_boxes)
+    , env_vertices(other.env_vertices)
+    , env_faces(other.env_faces)
+    , tree(other.tree)
+    , oriented_side(other.oriented_side)
+  {
+    *this = std::move(other);
+  }
```

Comment:
I do not understand: first you *copy* all member of `other` and then `*this = std::move(other)`?!

Reply:
copy-paste error from my initial try with the copy-constructor (that I delete now). Thanks!